TIDELIFT.rst

```diff
@@ -0,0 +1,57 @@
+========
+Tidelift
+========
+
+pytest is a member of `Tidelift`_. This document describes how the core team manages
+Tidelift-related activities.
+
+What is it
+==========
+
+Tidelift aims to make OpenSource sustainable by offering subscription to companies which rely
+on open source packages. This subscription allows it to pay maintainers of those Open Source
+packages to aid sustainability of the work.
+
+Funds
+=====
+
+It was decided in the `mailing list`_ that the Tidelift contribution will be split evenly between
+members of the `contributors team`_ interested in receiving funding.
```

Comment:
@flub in the ML you said "contributors". Did you mean [contributors](https://github.com/orgs/pytest-dev/teams/contributors/members) or [core contributors](https://github.com/orgs/pytest-dev/teams/core/members)? Just to make sure I'm getting this right. 😁

Reply:
This is what i meant :)